ndarray/src/main/java/org/tensorflow/ndarray/buffer/layout/TensorFormat.java

```diff
@@ -0,0 +1,30 @@
+package org.tensorflow.ndarray.buffer.layout;

```

Comment:
Thanks @sevarac , as we discussed briefly during our last session, this enum should probably be moved at the `tensorflow-core-api` or `tensorflow-framework` level. It would be helpful to know which one if you can provide some quick examples of usage. I think @JimClarke5 had some in mind too.

Reply:
Sounds good, in general everywhere where  public Options dataFormat(String dataFormat)  is used 
there should be now public Options dataFormat(TensorFormat dataFormat) 
which includes a bunch of classes mainly layers  https://github.com/tensorflow/java/search?q=dataFormat

Reply:
It is used in `losses.Losses` for `losses.CategoricalCrossentropy` and `metrics.CategoricalCrossentropy`.
```
  public static final int CHANNELS_LAST = -1;
  public static final int CHANNELS_FIRST = 1;
```

Once this PR is merged, I'll change the logic in losses and metrics.

Reply:
This could be done in the C++ op generator, by looking at any argument called `dataFormat`. Though I don't think this naming convention is enforced to the kernel developers, which might lead to mistakes. But there will be possible workarounds as well so I'm fine if you want to give a try making that change as well

Reply:
Hi @sevarac , so any chance that you can move this enum to a different location, as [I've suggested before](https://github.com/tensorflow/java/pull/191/files#r574243992)?

If I need to pick one, I'll suggest`tensorflow-framework` over `tensorflow-core-api`, what about under [`org.tensorflow.framework.utils`](https://github.com/tensorflow/java/tree/master/tensorflow-framework/src/main/java/org/tensorflow/framework/utils)?

Reply:
Also please don't forget to add the header notice in your file (like [this one](https://github.com/tensorflow/java/blob/master/tensorflow-framework/src/main/java/org/tensorflow/framework/utils/ShapeUtils.java), for example).